airbyte-integrations/connectors/source-freshdesk/source_freshdesk/schemas/scenario_automations.json

```diff
@@ -20,7 +20,7 @@
             "type": ["null", "string"]
           },
           "value": {
-            "type": ["null", "string"]
+            "type": ["null", "string", "integer"]
```

Comment:
As far as I can tell, this should just be `["null", "string"]`. Is there a reason this was changed?

Reply:
The value of some actions in the scenario automation can be an integer as well (that's what I found out during testing). I think the documentation mentions this, but I couldn't find it anymore.